.circleci/config.yml

```diff
@@ -1,10 +1,13 @@
 version: 2.1
+
 # reusable environment for all jobs
 executors:
   main-executor:
     working_directory: ~/sdk-component-adapter
     docker:
       - image: circleci/node:12.7.0
+        environment:
```

Comment:
Should we utilize a browser image from circle? https://circleci.com/docs/2.0/browser-testing/#prerequisites

Reply:
how is that beneficial for our case?

Reply:
I ask due to the title of the page linked: 

>This document describes common methods for running and debugging browser testing in your CircleCI config in the following sections:

Since we are adding browser testing, my question was should we use the tools provided.

Reply:
I see. Let's talk more about this.